index.js

```diff
@@ -1,5 +1,74 @@
 'use strict';
 
+var colorConvert = require('color-convert');
+var supportsColor = require('supports-color');
+
+supportsColor = supportsColor || {};
+supportsColor.level = supportsColor.level || 0;
+
+colorConvert.rgb.rgb = function rgb (rgb) {
```

Comment:
Converting something to itself isn't a conversion. It wouldn't make sense to have it in there, realistically speaking. This conversion makes it so we have to perform one less check when building up which converter we want to use.